README.md

```diff
@@ -205,7 +205,6 @@ Then refactor the application to use path-based functions rather than the defaul
 | `Router.currentUrl()` | `Router.currentPath()`  |
 | `Router.format()`     | `Router.formatPath()`   |
 | `Router.navigate()`   | `Router.navigatePath()` |
-| `Cmd.currentUrl()`    | `Cmd.currentPath()`     |
 | `Cmd.format()`        | `Cmd.formatPath()`      |
```

Comment:
There is no `Cmd.format()` function either, correct?

Reply:
This is the last thing, I promise then I will merge and publish :smile:

Reply:
Oh yeah, good catch! I forgot that function is pure (hence why I didn't make a cmd).